tests/jobs/localisation/test_methods.py

```diff
@@ -103,18 +102,11 @@ def create_parameter_from_decay_functions(method_name, grid):
     ny = grid.getNY()
     nz = grid.getNZ()
     filename = "tmp_" + method_name + ".grdecl"
-    if method_name == "const_gaussian_decay":
-        decay_obj = ConstGaussianDecay(
-            ref_pos,
-            main_range,
-            perp_range,
-            azimuth,
-            grid,
-            tapering_range,
-            use_cutoff,
-        )
-    elif method_name == "const_exponential_decay":
-        decay_obj = ConstExponentialDecay(
+    if method_name == "constant":
```

Comment:
Why this if cannot be part of `build_decay_object` function?

Reply:
Sorry for starting with "why":) I mean it in "I am curious why"-way

Reply:
Good question. `ConstantScalingFactor` is used only in `apply_constant` while the other decay-functions that get built in `build_decay_object` are for use in `apply_decay`.
This is not great and I think we can combine `apply_decay` and `apply_constant` into a single function, but perhaps in another PR.
What I really wanted to accomplish in this PR is to make `pylint` happy so that I can continue working on another PR that gets rid of `setup.py`.